packages/code-studio/src/settings/SettingsMenu.tsx

```diff
@@ -177,7 +195,25 @@ export class SettingsMenu extends Component<
                 </a>
               </div>
               <div className="app-settings-footer-item">
-                <div className="text-muted">Version {version}</div>
+                <div className="text-muted">UI Version {version}</div>
+              </div>
+              <div className="app-settings-footer-item">
+                <div className="text-muted">
+                  Engine Version {deephavenVersion}{' '}
+                  <div className="detailed-server-config">
+                    <FontAwesomeIcon icon={vsInfo} />
+                    <Tooltip>
```

Comment:
Interesting, the tooltip is hiding immediately after appearing in many cases... I think because it's just the right distance from the edge of the screen, where the tooltip arrow appears overtop of the item you are hovering over, so it triggers `mouseleave`.
I think _ideally_ we'd fix that on the Popper side - it should reposition the tooltip to be above because there isn't quite enough room on the right. But tracking that down might be tricky, and it's an upstream change.

Instead, just flag this tooltip as `interactive`. That way the tooltip will stay up even if they hover over the tooltip, and also has the added benefit (for this case) that you can select the text and copy it.

```suggestion
                    <Tooltip interactive>
```